keyboards/nullbitsco/nibble/keymaps/oled_bongocat/keymap.c

```diff
@@ -21,32 +21,75 @@
 
 
 enum layer_names {
-  _MA,
-  _FN
+  _BASE,
+  _VIA1,
+  _VIA2,
+  _VIA3
 };
 
+#define KC_DISC_MUTE KC_F23
+#define KC_DISC_DEAF KC_F24
+#define NUM_CUST_KEYCODES (_NUM_CUST_KCS - SAFE_RANGE)
+#define VIA_KEYCODE_RANGE 0x5F80
+
 enum custom_keycodes {
-    KC_CUST = SAFE_RANGE,
+  PROG = SAFE_RANGE,
+  DISC_MUTE,
+  DISC_DEAF,
+  SUPER_ALT_TAB,
+  _NUM_CUST_KCS,
 };
```

Comment:
This is way over complicated, IMO. 

If you use USER00-USER15, this is a lot simpler. Eg

```suggestion
enum custom_keycodes {
  PROG = USER00,
  DISC_MUTE,
  DISC_DEAF,
  SUPER_ALT_TAB,
  _NUM_CUST_KCS,
};
```